File: show.rb
class Show
  @@all_shows = []

  def initialize(title, location)
    # set attr value
    @title = title
    @location = location
    # save all objs into class arr
    @@all_shows << self
  end

  def self.formatShows() 
    puts 'info'
  end

  def self.all_shows
    @@all_shows
  end
  
end

show_obj = Show.new('Full House', 'San Fran')

File: tv_likes_frontend/src/show.js
class Show {
  static all_shows = []
  // class & instance method
  constructor(title, description) {
    // this the the new obj
    this.title = title
    this.description = description
    // console.log(this + `inside constructor`)
    Show.all_shows.push(this)
  }

  static formatShows(){
    console.log('info')
  }
}

// Show.all_shows = []

// console.log(this + `outside Show class`)
// instan... === MAKE NEW OBJ
// const showObj = new Show
// Show.formatShows()